src/cdomains/intDomain.ml

```diff
@@ -102,8 +102,9 @@ end
 
 module type IkindUnawareS =
 sig
-  include B with type int_t = int64
-  include Arith with type t:= t
+  type int_t
+  include B with type int_t := int_t
```

Comment:
Is this separation of `int_t` still necessary? I know we added it at some point, but then also undid something?

Reply:
It's still necessary, without it I'm getting: "Multiple definition of the type name int_t. Names must be unique in a given structure or signature." for `include B`.